grpcwebclient/impl.ts

```diff
@@ -71,6 +66,30 @@ function buildMetadata(req: ClientCompatRequest): Metadata {
     deadline.setMilliseconds(deadline.getMilliseconds() + req.timeoutMs);
     metadata.deadline = deadline.getTime().toString();
   }
+
+  switch (req.compression) {
+    case Compression.GZIP:
+      metadata["content-encoding"] = "gzip";
+      break;
+    case Compression.ZSTD:
+      metadata["content-encoding"] = "zstd";
+      break;
+    case Compression.BR:
+      metadata["content-encoding"] = "br";
+      break;
+    case Compression.DEFLATE:
+      metadata["content-encoding"] = "deflate";
+      break;
+    case Compression.SNAPPY:
+      metadata["content-encoding"] = "snappy";
+      break;
+    case Compression.IDENTITY:
+      metadata["content-encoding"] = "identity";
+      break;
+    default:
+    // Do nothing
+  }
```

Comment:
Let's back this out. This is full-body compression, which is not the expected way to do compression with the gRPC / gRPC-Web protocol.